pythonforandroid/recipes/hostpython3/__init__.py

```diff
@@ -35,6 +35,8 @@ class Hostpython3Recipe(Recipe):
     '''The default url to download our host python recipe. This url will
     change depending on the python version set in attribute :attr:`version`.'''
 
+    patches = ('patches/pyconfig_detection.patch',)
```

Comment:
is the new patch working/needed for Python 3.7?

Reply:
Good question, I've only verified for 3.8 to be honest. I can apply it to 3.8 only to be safe